source/Applications/Advanced/PoseConversions/PoseConversions.cs

```diff
@@ -0,0 +1,325 @@
+
+            Console.WriteLine("RotationMatrix:\n" + matrixToString(rotationMatrix));
+            Console.WriteLine("TranslationVector:\n" + matrixToString(translationVector.Transpose()));
+
+            /*
+             * Convert from Rotation Matrix (Zivid) to other representations of orientation (Robot)
+             */
+            printHeader("Convert from Zivid (Rotation Matrix) to Robot");
+            var axisAngle = rotationMatrixToAngleAxis(rotationMatrix);
+            Console.WriteLine("AxisAngle:\n" + matrixToString(axisAngle.Item1.Transpose()) + ", " + String.Format(" {0:G4} ", axisAngle.Item2));
+            var rotationVector = axisAngle.Item1 * axisAngle.Item2;
+            Console.WriteLine("Rotation Vector:\n" + matrixToString(rotationVector.Transpose()));
+            var quaternion = rotationMatrixToQuaternion(rotationMatrix);
+            Console.WriteLine("Quaternion:\n" + matrixToString(quaternion.Transpose()));
+            var rpyList = rotationMatrixToRollPitchYawList(rotationMatrix);
+
+            /*
+             * Convert to Rotation Matrix (Zivid) from other representations of orientation (Robot)
+             */
+            printHeader("Convert from Robot to Zivid (Rotation matrix)");
+            var rotationMatrixFromAxisAngle = axisAngleToRotationMatrix(axisAngle);
+            Console.WriteLine("Rotation Matrix from Axis Angle:\n" + matrixToString(rotationMatrixFromAxisAngle));
+
+            var rotationMatrixFromRotationVector = rotationVectorToRotationMatrix(rotationVector);
+            Console.WriteLine("Rotation Matrix from Rotation Vector:\n" + matrixToString(rotationMatrixFromRotationVector));
+            var rotationMatrixFromQuaternion = quaternionToRotationMatrix(quaternion);
+            Console.WriteLine("Rotation Matrix from Quaternion:\n" + matrixToString(rotationMatrixFromQuaternion));
+
+            rollPitchYawListToRotationMatrix(rpyList);
+
+
+            Console.WriteLine("Rotation from print:\n" + matrixToString(rotationMatrix));
+
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine("Error: {0}", ex.Message);
+            Environment.ExitCode = 1;
+        }
+    }
+    enum RotationConvention
+    {
+        zyxIntrinsic,
+        xyzExtrinsic,
+        xyzIntrinsic,
+        zyxExtrinsic
+    };
+    const int nrOfRotationConventions = 4;
+    static string conventionToString(RotationConvention convention)
+    {
+        switch (convention)
+        {
+            case RotationConvention.xyzIntrinsic: return "xyzIntrinsic";
+            case RotationConvention.xyzExtrinsic: return "xyzExtrinsic";
+            case RotationConvention.zyxIntrinsic: return "zyxIntrinsic";
+            case RotationConvention.zyxExtrinsic: return "zyxExtrinsic";
+        }
+        throw new ArgumentException("Invalid RotationConvention");
+    }
+
+    static Matrix<double> scew(Matrix<double> vector)
```

Comment:
Shouldn't this be called skew? https://en.wikipedia.org/wiki/Skew-symmetric_matrix

Reply:
Wow that was embarrassing... Yes it should be skew